app/models/legal_aid_application.rb

```diff
@@ -228,6 +229,8 @@ def add_benefit_check_result
   end
 
   def applicant_receives_benefit?
+    return true if dwp_override
+
```

Comment:
yep

Reply:
My immediate reaction was yes, but looking more closely, I think it's better as it is.  I am a believer in methods with a question mark only returning `true` or `false`.  Your suggestion would return nil if there was no `dwp_override` AND no `benefit_check_result`.

It could be 

   ```dwp_override || (benefit_check_result&.positive? || false)```

but I think that is harder to understand.

Reply:
fair enough 👍